Add note about raw and verbatim tags to handbook. (#1159)
GoogleChrome/web.dev: src/site/content/en/handbook/markup-code/index.md

</label>
```

> ## Escaping code

Reviewer: ```suggestion
## Escaping special characters
```

> If you need to use double `{` braces in your code examples you will need to

Reviewer: ```suggestion
To keep the parser from interpreting special characters in your code samples (for example, `{{`), escape them using the `{% raw %}{% verbatim %}` shortcode combination:
```
Trying to generalize and make explicit that both shortcodes are needed. Not sure "parser" is the right word. Feel free to just fix and publish if it isn't.

Author: I avoided using double curly braces in the example itself because we have to escape them :D

Reviewer: Well, <facepalm> for not escaping it (sorry), but I do think it's clearer to just show the double curly braces than write out the phrase. But not a big deal either way.

Author: Totally agree. Unfortunately because we publish the handbook to devsite I don't think I'm able to show the double curlies without also showing the unescaped `{% verbatim %}` around them. I thought that might confuse folks so I went with "double `{`"

> escape them:

Reviewer: ```suggestion
```

```html
{% raw %}{% raw %}{% verbatim %}{{ item.num | calculate }}{% endverbatim %}{% endraw %}{% endraw %}
```

> The `{% raw %}{% verbatim %}{% endraw %}` tags will be visible when you stage

Reviewer: ```suggestion
{% Aside %}
The `{% raw %}{% verbatim %}{% endraw %}` shortcodes will be visible when you stage
```
Are we referring to these as tags or shortcodes? If tags, I need to fix elsewhere in the guide.

Author: Not sure if it's too much information but...

Lower case things like `{% raw %}` are Nunjucks "blocks".
Uppercase things like `{% Aside %}` are eleventy "shortcodes".

Reviewer: Ah-ha. "Tags" it is :)

> locally but not in production.

Reviewer: ```suggestion
locally but not in production.
{% endAside %}
```
